<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmoticonsManager</class>
 <widget class="QWidget" name="EmoticonsManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emoticons Manager</string>
  </property>
  <layout class="QGridLayout">
   <item row="4" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="btEdit">
     <property name="toolTip">
      <string>Modify the selected emoticon icon or text </string>
     </property>
     <property name="whatsThis">
      <string>Edit the selected emoticon to change its icon or its text</string>
     </property>
     <property name="text">
      <string>Edit...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btAdd">
     <property name="toolTip">
      <string>Create a new emoticon</string>
     </property>
     <property name="whatsThis">
      <string>Create a new emoticon by assigning it an icon and some text</string>
     </property>
     <property name="text">
      <string>Add...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2" colspan="5">
    <widget class="QListWidget" name="emoList">
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QListWidget" name="themeList">
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btRemoveTheme">
     <property name="toolTip">
      <string>Remove the selected theme from your disk</string>
     </property>
     <property name="whatsThis">
      <string>This will remove the selected theme from your disk.</string>
     </property>
     <property name="text">
      <string>Remove Theme</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="btRemoveEmoticon">
     <property name="toolTip">
      <string>Remove the selected emoticon</string>
     </property>
     <property name="whatsThis">
      <string>Remove the selected emoticon from your disk</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btNew">
     <property name="toolTip">
      <string>Design a new emoticon theme</string>
     </property>
     <property name="whatsThis">
      <string>Start a new theme by assigning it a name. Then use the Add button on the right to add emoticons to this theme.</string>
     </property>
     <property name="text">
      <string>New Theme</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="4">
    <widget class="QCheckBox" name="cbStrict">
     <property name="text">
      <string>Require spaces around emoticons</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btGetNew">
     <property name="toolTip">
      <string>Get new icon themes from the Internet</string>
     </property>
     <property name="whatsThis">
      <string>You need to be connected to the Internet to use this action. A dialog will display a list of emoticon themes from the http://www.kde-look.org website. Clicking the Install button associated with a theme will install this theme locally.</string>
     </property>
     <property name="text">
      <string>Get new Theme</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btInstall">
     <property name="toolTip">
      <string>Install a theme archive file you already have locally</string>
     </property>
     <property name="whatsThis">
      <string>If you already have an emoticon theme archive locally, this button will unpack it and make it available for KDE applications</string>
     </property>
     <property name="text">
      <string>Install from File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
